src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -33,6 +33,7 @@ static const Duration AnimationDuration = DurationHelper::FromTimeSpan(winrt::Wi
 
 winrt::Windows::UI::Xaml::Media::SolidColorBrush Pane::s_focusedBorderBrush = { nullptr };
 winrt::Windows::UI::Xaml::Media::SolidColorBrush Pane::s_unfocusedBorderBrush = { nullptr };
+winrt::Windows::Media::Playback::MediaPlayer Pane::s_bellPlayer = { nullptr };

```

Comment:
ALRIGHT!

In 1.13.306x, a static MediaPlayer is getting torn down on shutdown and causing a crash!

We need to either leak this intentionally (throw an AddRef at it, mark it BODGY) or make it *not* static lifetime.

Reply:
As long as it isn't weird for a single process to have like.. 40 media players. Any guidance in the API docs?

Reply:
@DHowett how did you ID the crash on teardown as the `MediaPlayer`? I'm seeing the crash, but that's up in `Microsoft.Terminal.Remoting.dll!winrt::Microsoft::Terminal::Remoting::implementation::Monarch::SignalClose(const unsigned __int64 peasantId) Line 212`, which is pretty far away... Ah wait I see, you step over that and _then_ we hit the frames in `Windows.Media.Playback.MediaPlayer.dll!00007ff853c6f9a9()`. Gotcha. Yep, let's try the TerminalPage thing.

Reply:
you marked this one as resolved but we're still using the global static media player -- on purpose?

Reply:
see 93937533bc00c53311867663a0a3a94d73253862

Reply:
oh heck. i purged that from my brain over the holidays because I wanted to be happy.

Reply:
we work on the console, we're not allowed to be happy 😝